packages/frontend/next.config.js

```diff
@@ -1,13 +1,20 @@
 /** @type {import('next').NextConfig} */
+const FilterWarningsPlugin = require('webpack-filter-warnings-plugin');
+
 const nextConfig = {
   reactStrictMode: true,
   transpilePackages: ['react-syntax-highlighter', 'swagger-client', 'swagger-ui-react'],
   webpack: (config) => {
     config.resolve.fallback = { fs: false };
     config.experiments = { asyncWebAssembly: true, layers: true };
-
+    config.plugins.push(
+      new FilterWarningsPlugin({
+        exclude: [/Critical dependency/],
```

Comment:
it seems like the specific warning we want to suppress is around dependencies that are expressions. Is it possible to scope the exclude statement to that?

I can't reproduce the warning, but maybe something like `/Critical dependency  the request of a dependency is an expression/` is sufficient to suppress the warning that's annoying us

Wary of over-suppressing.

Reply:
Good point, I'll test that out. Don't want to oversurpress.

Reply:
Fixed by [1492c97](https://github.com/personaelabs/nym/pull/67/commits/1492c97a1504ab36a95071f0cec3dbe1af02e448)